pkg/ui/src/views/statements/statementsPage.tsx

```diff
@@ -59,7 +69,7 @@ class StatementsPage extends React.Component<StatementsPageProps & RouteProps, S
     super(props);
     this.state = {
       sortSetting: {
-        sortKey: 1,
+        sortKey: 2,
```

Comment:
Honestly, I think latency should be the default sort (it's what people care about; I don't think "total time" is that meaningful…)